pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -77,6 +77,8 @@ type PreEnqueueCheck func(pod *v1.Pod) bool
 type SchedulingQueue interface {
 	framework.PodNominator
 	Add(pod *v1.Pod) error
+	// MoveUpPods moves given pods to activeQ in a best-efforts manner.
```

Comment:
explain what's the `given pods`, where it comes from and the rationale behind the scenes for better understanding of the motivation.

Reply:
updated.